backend-base/src/main/java/org/eclipse/che/incubator/workspace/telemetry/base/DefaultAnalyticsManager.java

```diff
@@ -42,8 +42,17 @@ public void onActivity() {
   public void onEvent(AnalyticsEvent event, String ownerId, String ip,
       String userAgent, String resolution, Map<String, Object> properties) {
     LOG.info("Event triggered by user {} in {} from ip {} on agent {} :\n{}\nwith resolution: {}\nwith properties:\n{}", getUserId(), ownerId, ip, userAgent, event, resolution, properties);
+    lastEvent = event;
+    lastEventTime = System.currentTimeMillis();
+    lastIp = ip;
+    lastUserAgent = userAgent;
+    lastResolution = resolution;
+    lastEventProperties = properties;
   }
 
+  @Override
+  public void increaseDuration(AnalyticsEvent event, Map<String, Object> properties) {}
```

Comment:
should be empty by design?

Reply:
would be also nice to add a java doc

Reply:
yeah this is empty by design.  The idea is that if you have a telemetry backend with a notion of duration for events, you would use this method to "turn the knob" on the duration of the event.

Reply:
it is a no-op in the woopra code as well because duration is handled transparently